comet/scrapers/manager.py

```
@@ -131,6 +144,26 @@ async def scrape_torrents(
+                    torrents_data = orjson.loads(cached_data) if isinstance(cached_data, str) else cached_data
+                    for info_hash, torrent_data in torrents_data.items():
+                        self.torrents[info_hash] = {
+                            "fileIndex": torrent_data["fileIndex"],
+                            "title": torrent_data["title"],
+                            "seeders": torrent_data["seeders"],
+                            "size": torrent_data["size"],
+                            "tracker": torrent_data["tracker"],
+                            "sources": torrent_data["sources"],
+                            "parsed": ParsedData(**torrent_data["parsed"]),
+                        }
+                    return
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)